src/ReadLine/ReadLine.xproj

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Let's just use `project.json` for now. This project will migrate to `csproj` when the dotnet cli comes out of preview